VictoriaAlbert/VictoriaAlbert/ViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Thinley Dorjee on 11/10/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var detailImage: UIImageView!
+    @IBOutlet weak var homeLabel: UILabel!
+    var imageEndPoint = "http://media.vam.ac.uk/media/thira/collection_images/2006AM/2006AM6763.jpg"
```

Comment:
I see you couldn't figure out how to build the image urls by string manipulations. This was a good workaround so you can work on the rest of the app. But you should also work on your string manipulation because it's important.